cameo/cli/controllers.py

```diff
@@ -108,14 +108,17 @@ def default(self):
             view = SequentialView()
 
         design.debug = self.app.pargs.test
+        try:
+            results = design(product=product, hosts=_hosts,
+                             view=view, aerobic=not self.app.pargs.anaerobic)
 
-        results = design(product=product, hosts=_hosts,
-                         view=view, aerobic=not self.app.pargs.anaerobic)
+            results['heterologous_pathway'] = results.heterologous_pathway.apply(str)
+            results['manipulations'] = results.manipulations.apply(str)
 
-        results['heterologous_pathway'] = results.heterologous_pathway.apply(str)
-        results['manipulations'] = results.manipulations.apply(str)
-
-        OUTPUT_WRITER[output_format](results, output)
+            OUTPUT_WRITER[output_format](results, output)
+        except KeyError as e:
```

Comment:
Not here, at this point we know what raised KeyError there, because we raised it. The CLI should fail gracefully without stack trace for exceptions, just a message explaining the user  why it didn't work.